src/controllers/discussion-controller.js

```diff
@@ -0,0 +1,168 @@
+const Discussions = require('../models/discussions');
+const Accounts = require('../models/accounts');
+const {StatusCodes} = require ('http-status-codes');
+const mongoose = require('mongoose');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const mongoose = require("mongoose");
```